src/window_keyboard.cpp

```diff
@@ -16,20 +16,21 @@
  */
 
 #include <string>
+#include <algorithm>
 
 #include "window_keyboard.h"
 #include "game_system.h"
 #include "input.h"
 #include "bitmap.h"
 #include "font.h"
 
-const char* const Window_Keyboard::DONE = "<Done>";
-const char* const Window_Keyboard::SPACE = "SPACE";
-const char* const Window_Keyboard::NEXT_PAGE = "<Page>";
-const char* const Window_Keyboard::DONE_JP = "<決定>";
-const char* const Window_Keyboard::DONE_RU = "<OK>";
-const char* const Window_Keyboard::DONE_KO = "<결정>";
-const char* const Window_Keyboard::DONE_ZH_CN = "<确定>";
+const char* Window_Keyboard::DONE = "<Done>";
+const char* Window_Keyboard::SPACE = "SPACE";
+const char* Window_Keyboard::NEXT_PAGE = "<Page>";
+const char* Window_Keyboard::DONE_JP = "<決定>";
+const char* Window_Keyboard::DONE_RU = "<OK>";
+const char* Window_Keyboard::DONE_KO = "<결정>";
+const char* Window_Keyboard::DONE_ZH_CN = "<确定>";
```

Comment:
How about "constexpr char Window_Keyboard::DONE[] =" ;)?

Reply:
This does not work. Either I get `-Wwriteable-strings` warnings or the build errors out with
`error: non-const static data member must be initialized out of line` or
`error: declaration of constexpr static data member 'SPACE' requires an initializer` or
`error: redefinition of 'SPACE' with a different type: 'char *const' vs 'const char *'`, depending on how i modif/move them. 🤕

Reply:
o.O okay leave it as is.